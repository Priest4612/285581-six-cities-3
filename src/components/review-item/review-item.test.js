import React from 'react';
import renderer from 'react-test-renderer';

import ReviewItem from './review-item.jsx';


const review = {
  id: 2,
  authorAvatar: `angelina`,
  authorName: `Angelina`,
  rating: 3.8,
  description: `123 3rfg432 54t4566 56457 5436er5 6456`,
  dateTime: `23-02-2020`,
  idOffer: 101,
};


describe(`Checked OfferDetails`, () => {
  it(`Render OfferDetails`, () => {
    const tree = renderer
      .create(<ReviewItem
        review={review}
      />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
