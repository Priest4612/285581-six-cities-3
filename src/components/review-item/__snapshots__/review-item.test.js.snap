// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checked OfferDetails Render OfferDetails 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Reviews avatar"
        className="reviews__avatar user__avatar"
        height="54"
        src="img/avatar-angelina.jpg"
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      Angelina
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "76%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          3.8
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      123 3rfg432 54t4566 56457 5436er5 6456
    </p>
    <time
      className="reviews__time"
      dateTime="23-02-2020"
    >
      23-02-2020
    </time>
  </div>
</li>
`;
